assets/js/components/Work/Tabs/Administrative.jsx

```diff
@@ -51,13 +90,24 @@ const WorkTabsAdministrative = ({ work }) => {
             </div>
 
             <div className="field">
-              <label className="label">Collection</label>
+              <label className="label">
+                Collection <UITagNotYetSupported label="Work in Progress" />
+              </label>
               <div className="control">
                 {isEditing ? (
                   <div className="select">
-                    <select ref={register} name="collection">
-                      <option>Select dropdown</option>
-                      <option>With options</option>
+                    <select
+                      ref={register}
+                      name="collection"
+                      defaultValue={work.collection ? work.collection.id : ""}
```

Comment:
Mind destructuring any field present in the `work` object that the code > 1 times?   I'd put this at the top of the component definition, around line 17 ish....something like:

```
const { id, administrativeMetadata , collection, .....} = work;
```